test/unit/lib/plugins/aws/deploy-function.test.js

```diff
@@ -566,21 +567,121 @@ describe('test/unit/lib/plugins/aws/deployFunction.test.js', () => {
   });
 
   it('should update function configuration if configuration changed', async () => {
-    await runServerless({
+    const runServerlessConfig = {
```

Comment:
It would be nice to have `runSeverless` run in dedicated `describe` block in `before` clause and then validate each case with individual `it`

Reply:
I've just realized that we cannot have single `runServerless` run as we're testing `deploy function` which naturally will just deploy single function at once 🤦 

In light of that, what you had before was way better (each `it` with individual `runServerless` run),

My initial point was to invoke `runServerless` just one time, and confirm on outcome of different functions in different `it`, but obviously that can work only for service deployments and not single function deployments

Reply:
Reverted 😎